examples/apps/generic_client/generic_client.py

```diff
@@ -190,6 +190,18 @@ def _lookup_first_worker(worker_registry, jrpc_req_id):
 
 
 def _do_worker_verification(worker_obj):
+    encryption_key_signature = worker_obj.encryption_key_signature
```

Comment:
Let's verify encryption key signature only if `worker_obj.encryption_key_signature` is not None.

Reply:
Done